### PR TITLE
Fixed: Artist and Album Posters show placeholder instead of image

### LINKED ISSUE #169: Artist poster has issues when one poster missing (Web UI)
in web ui, some artists occasionally only show gray lidarr icon for image and the image can come and go when the artist is refreshed:

2017-12-28 22_14_51-lidarr

furthermore artist images on adjacent pages are affected when paging through artists using the <- & -> arrow buttons.  E.g. Muse page usually shows as

2017-12-28 22_16_56-muse - lidarr

whereas, when paging right from Moon Taxi (artist with default gray lidarr icon shown at top), you get:

2017-12-28 22_18_50-muse - lidarr
 

and paging from the left a good artist image Michael Kiwanuka showing for Moon Taxi lol:
2017-12-28 22_19_59-moon taxi - lidarr

I'm only posting this because deep down I'm hoping it's related to issue #168 and album images not being handled/embedded properly although I know that's slim to none.

### PR DESCRIPTION
**Pull request by majora2007**
#### Database Migration
NO

#### Description
Fixes a bug in which Album Posters sometimes remain blank when previously they were loaded.

#### Issues Fixed or Closed by this PR

* Fixes #169

### CODE REVIEW

**Comment by majora2007** [comment]
I have now :)

**Comment by Qstick** [comment]
Don't forget same change in ArtistBanner.js, I'll give it a test after work. I'm curious about the note that @markus101 had in there. Appears they where not setting isLoaded for a reason.

**Comment by majora2007** [comment]
Sure, I will update banner as well. I was curious about Markus' note as well, but I observed that when we loaded an album previously loaded from an error stated lazy load, the isLoaded flag was never true and hence the placeholder was there. Hence why setting isLoaded after fetching the next, non-error cover fixes the issue.

**Comment by markus101** [comment]
I forget exactly offhand, but it may have been due to the virtual list not updating when an item was removed, whether that had a poster or not I forget. Worth checking the main page when artist/album art is shown.

**Comment by Qstick** [comment]
Looks good, and all working as expected. I pushed up a small fix to banner that I missed on a previous update of poster.

File: frontend/src/Album/AlbumCover.js [hunk 1]
```diff
@@ -64,10 +64,9 @@ class AlbumCover extends Component {
       this.setState({
         cover: nextcover,
         coverUrl: getCoverUrl(nextcover, pixelRatio * size),
-        hasError: false
-        // Don't reset isLoaded, as we want to immediately try to
-        // show the new image, whether an image was shown previously
-        // or the placeholder was shown.
+        hasError: false,
+        isLoaded: true,
+        lazy: true
```
Comment (Qstick) [inline frontend/src/Album/AlbumCover.js]: is `lazy: true` needed here? it's not used on ArtistPoster or Banner, looks like default value is true.

**Comment by majora2007** [comment]
No it isn't.

**Pull request merged**